css-fonts/styles.css

```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: 'Oswald', sans-serif;
```

Comment:
You should be using 2 separate fonts. One for the headings, and one for the rest of the content

Reply:
My bad Shawn!
![234357](https://user-images.githubusercontent.com/122063836/214682535-5de3fe79-adea-4617-bdc6-7baab0c4aa95.PNG)